usr/src/uts/common/io/vnd/vnd.c

```diff
@@ -2529,8 +2530,10 @@ vnd_st_promiscoff(vnd_str_t *vsp)
 		return;
 	}
 
+	/* We're successful, free the message we popped. */
+	freemsg(mp);
 	if (prim == DL_ERROR_ACK) {
-		cmn_err(CE_WARN, "!failed to disable promiscuos mode during "
+		cmn_err(CE_WARN, "!failed to disable promiscous mode during "
```

Comment:
promiscous -> promiscuous

Reply:
I only half-fixed the original spelling mistake, huh?  Update coming.